cvl/internal/util/util.go

```diff
@@ -253,3 +260,184 @@ func SkipSemanticValidation() bool {
 
 	return false
 }
+
+//Function to read Redis DB configuration from file.
+//In absence of the file, it uses default config for CONFIG_DB
+//so that CVL UT will pass in development environment.
+func dbCfgInit() {
+	defaultDBConfig := `{
```

Comment:
Can we  use [sonic_db_config/db_config.go](https://github.com/Azure/sonic-telemetry/blob/master/sonic_db_config/db_config.go) ?

We could move that to here and update as needed. So we can have all the APIs that parse and use this file, in one place.

Reply:
Thought of so, but wanted keep CVL as independent of telemetry package.

Are you suggesting to import "sonic-telemetry/dbconfig" in this file and create a wrapper layer for any additional logic ?

Reply:
Better to keep the common stuff in one place, so everyone shares the updated code.

Reply:
Renuka, we cannot do that as telemetry code today is dependent on mgmt-common and CVL resides inside the mgmt-common. We need to do the other way around all the telemetry DB access stuff move to mgmt-common to make everything reside in a common place.